stock_prices/stock_prices.py

```diff
@@ -1,9 +1,18 @@
 #!/usr/bin/python
+# pull request
 
 import argparse
+import math
 
 def find_max_profit(prices):
-  pass
+  temp = math.inf * -1
+  for x in range(0,len(prices)):
+    for y in range(x+1, len(prices)):
+      difference = prices [y] - prices[x]
+      if difference > temp:
+        temp = difference
+
+  return temp
```

Comment:
Well done Noah! Very simple, yet effective approach. With a little bit of work you might be able to cut out one of your `for` loops to improve performance. How might you go about this?